core/templates/pages/exploration-editor-page/editor-tab/exploration-editor-tab.component.ts

```diff
@@ -380,12 +389,16 @@ export class ExplorationEditorTabComponent
           this.versionHistoryBackendApiService.fetchStateVersionHistoryAsync(
             this.contextService.getExplorationId(), stateData.name,
             this.versionHistoryService.getLatestVersionOfExploration()
-          ).then((response: StateVersionHistoryResponse) => {
-            this.versionHistoryService.insertStateVersionHistoryData(
-              response.lastEditedVersionNumber,
-              response.stateInPreviousVersion,
-              response.lastEditedCommitterUsername
-            );
+          ).then((response) => {
+            if (response !== null) {
+              this.versionHistoryService.insertStateVersionHistoryData(
+                response.lastEditedVersionNumber,
+                response.stateInPreviousVersion,
+                response.lastEditedCommitterUsername
+              );
+            } else {
+              this.validationErrorIsShown = true;
+            }
```

Comment:
Is there a test for if and else condition?

Reply:
Added it now.